app/controllers/api/messages_controller.rb

```diff
@@ -1,9 +1,9 @@
 class Api::MessagesController < Api::ApplicationController
   def index
     @messages = Message.find_by(id: rand(1..Message.count))
-    response.headers['Access-Control-Allow-Origin'] = 'http://127.0.0.1:3000'
-    response.headers['Access-Control-Allow-Methods'] = 'GET'
-    response.headers['Access-Control-Allow-Headers'] = 'Content-Type'
+    # response.headers['Access-Control-Allow-Origin'] = 'http://127.0.0.1:3000'
+    # response.headers['Access-Control-Allow-Methods'] = 'GET'
+    # response.headers['Access-Control-Allow-Headers'] = 'Content-Type'
```

Comment:
- [ ] Kindly remove the commented code to make your codebase clean and to follow Ruby best practices :+1: